### DISCUSSION TITLE
v1.24.1 release

### DISCUSSION THREAD

**Discussion by andyzhangx** [kubernetes-sigs/blob-csi-driver#1348] [Announcements]
## What's Changed
* [release-1.24] feat: allow multiple subnets in updating service endpoints by @k8s-infra-cherrypick-robot in https://github.com/kubernetes-sigs/blob-csi-driver/pull/1260
* [release-1.24] fix: ensure azure cloud config could be loaded from secret by @k8s-infra-cherrypick-robot in https://github.com/kubernetes-sigs/blob-csi-driver/pull/1266
* [release-1.24] fix: nfs volume created in storage account that has blobfuse volume by @k8s-infra-cherrypick-robot in https://github.com/kubernetes-sigs/blob-csi-driver/pull/1273
* [release-1.24] feat: provide nfsv3 protocol to enforce nfs mount by @k8s-infra-cherrypick-robot in https://github.com/kubernetes-sigs/blob-csi-driver/pull/1277
* [release-1.24] test: ignore azcopy CVE-2024-24786 by @k8s-infra-cherrypick-robot in https://github.com/kubernetes-sigs/blob-csi-driver/pull/1280
* [release-1.24] fix: fix azcopy exec timeout func by @k8s-infra-cherrypick-robot in https://github.com/kubernetes-sigs/blob-csi-driver/pull/1285
* [release-1.24] feat: upgrade azcopy version to 10.23.0 for volume clone feature by @k8s-infra-cherrypick-robot in https://github.com/kubernetes-sigs/blob-csi-driver/pull/1287
* [release-1.24] fix: nfs mount stuck issue by @k8s-infra-cherrypick-robot in https://github.com/kubernetes-sigs/blob-csi-driver/pull/1292
* [release-1.24] chore: only use sas token for azcopy when useDataPlaneAPI is used by @k8s-infra-cherrypick-robot in https://github.com/kubernetes-sigs/blob-csi-driver/pull/1296
* [release-1.24] cleanup: rename WaitUntilTimeout func by @k8s-infra-cherrypick-robot in https://github.com/kubernetes-sigs/blob-csi-driver/pull/1299
* [release-1.24] chore: refine refine private endpoint to reduce error messages by @andyzhangx in https://github.com/kubernetes-sigs/blob-csi-driver/pull/1301
* [release-1.24] chore: refine sastoken cache by @k8s-infra-cherrypick-robot in https://github.com/kubernetes-sigs/blob-csi-driver/pull/1304
* [release-1.24] fix: strip service account token by @k8s-infra-cherrypick-robot in https://github.com/kubernetes-sigs/blob-csi-driver/pull/1310
* [release-1.24] fix: workload identity did not work by @k8s-infra-cherrypick-robot in https://github.com/kubernetes-sigs/blob-csi-driver/pull/1319
* [release-1.24] chore(deps): bump build-image/debian-base from bookworm-v1.0.1 to bookworm-v1.0.2 in /pkg/blobplugin by @k8s-infra-cherrypick-robot in https://github.com/kubernetes-sigs/blob-csi-driver/pull/1322
* [release-1.24] fix: enable http with track2 sdk by @andyzhangx in https://github.com/kubernetes-sigs/blob-csi-driver/pull/1328
* [release-1.24] feat: use new restricted liveness probe endpoint by @andyzhangx in https://github.com/kubernetes-sigs/blob-csi-driver/pull/1337
* [release-1.24] feat: upgrade to azcopy v10.24.0 for volume clone feature by @k8s-infra-cherrypick-robot in https://github.com/kubernetes-sigs/blob-csi-driver/pull/1335
* [release-1.24] feat: support setting kubelet path for blobfuse-proxy endpoint by @cvvz in https://github.com/kubernetes-sigs/blob-csi-driver/pull/1342
* [release-1.24] fix: allow http auth in volume clone by @k8s-infra-cherrypick-robot in https://github.com/kubernetes-sigs/blob-csi-driver/pull/1345
* doc: cut v1.24.1 release by @andyzhangx in https://github.com/kubernetes-sigs/blob-csi-driver/pull/1346

**Full Changelog**: https://github.com/kubernetes-sigs/blob-csi-driver/compare/v1.24.0...v1.24.1

<hr /><em>This discussion was created from the release <a href='https://github.com/kubernetes-sigs/blob-csi-driver/releases/tag/v1.24.1'>v1.24.1 release</a>.</em>

**Comment by mkemmerz** [comment]
Will the 1.24.1 also be offered as a Helm Chart version? Currently only seeing the 1.24.0.

**Reply by andyzhangx** [reply]
pls update the helm chart repo, it's done

**Reply by mkemmerz** [reply]
great job, thanks! :)
